### ISSUE TITLE
description language not being replaced.

### ISSUE THREAD

**Issue by karneaud** [Regulus343/ActivityLog#28]
I'm using the latest version of your package 

```
"regulus/activity-log": "^0.6.7"
```

I've set the replacement prefixes to default to "labels" and create a "labels.php" language file with the respective keys 

and when language_key is to true when passing the params

```
return [
        'contentId' => $model->id,
        'contentType'=> $model->getTable(),
        'action'=> $action,
        'updated' => ($action === 'update'),
        'developer' => 0,
        'description' => [
          'updated_item', [
            'user' => auth()->user()->name,
            'item' => 'SPAL|labels.claims'
          ]
        ]
      ];
```

I get the following record entry

>  id, user_id, content_type, content_id, action, description, details, data, language_key, public, developer, ip_address, user_agent, created_at, updated_at
> '9', '12', 'claims', '11', 'Update', '[\"updated_item\",{\"user\":\"zmayer_distributor\",\"item\":\"SPAL|labels.claims\"}]', NULL, NULL, '1', '1', '0', '192.168.10.10', 'Mozilla/5.0 (Windows NT 6.3; WOW64; rv:51.0) Gecko/20100101 Firefox/51.0', '2017-03-01 22:40:56', '2017-03-01 22:40:56'

Is this how the content for the description field should look like or am I  missing something?

**Comment by Regulus343** [comment]
Yes, it saves the literal keys and formatting data to the database and then formats it when you use `$activity->getDescription()` and `$activity->getDetails()` on a record. That way, you can revise your language files and old records will still reflect the recent language file updates. Please let me know if `getDescription()` doesn't work for you.

**Comment by karneaud** [comment]
+1 on this.

**Comment by Regulus343** [comment]
Thanks for the great description of your issue! I have reproduced and corrected this issue for v0.6.8, but I'll leave the issue open to give you a chance to confirm that your issue is solved.
